src/nnue/nnue_feature_transformer.h

```diff
@@ -392,6 +392,9 @@ class FeatureTransformer {
                 BiasType sum0 = accumulation[static_cast<int>(perspectives[p])][j + 0];
                 BiasType sum1 =
                   accumulation[static_cast<int>(perspectives[p])][j + HalfDimensions / 2];
+              
+              // Read more about why the "*2" is necessary here:
+              // https://github.com/official-stockfish/Stockfish/pull/5282#issuecomment-2131582943
```

Comment:
Not a fan of linking back to old github history.  If it can't be explained simply inline I'd prefer leaving it as is.

Reply:
Ok, will write the direct explanation.